std/portable.json

```diff
@@ -6,6 +6,7 @@
     "allowJs": true,
     "downlevelIteration": true,
     "preserveConstEnums": true,
+    "skipLibCheck": true,
```

Comment:
Not sure this is wanted, as both JS and AS libs are merged?

Reply:
It should speedup typechecking

Reply:
I believe, though, that the end user should decide, not the portable base config, hmm.

Reply:
Ok, I'll move it to std/tsconfig